smaretski/03_HTML/task_01_forms.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>  
+<HTML> 
+    <HEAD>
+        <META charset="utf-8">
+        <TITLE>Forms</TITLE> 
+    </HEAD>
+    <BODY> 
+        <table width="700" border="0">
+		<tr>
+		<th align="left" valign="top">
+		<b>Простая форма поиска с вариантами:</b>
+		<p><form id="my_form_1" name="search_1" method="get" action="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+		<legend>Строка поиска</legend>
+		<input type="text" name="search">
+		<input type="submit" value="Искать">
+		<br>
+		<label> <input type="radio" name="radio_name_1" value="site" id="checkbox_group_1" checked>на сайте
+		</label>
+		<label> <input type="radio" name="radio_name_1" value="google" id="checkbox_group_2" checked>в google
+		</label>
+		</fieldset>
+		</form>
+		</p>
+		</th>
+		<th align="left" valign="top">
+		<b>Простая форма поиска с вариантами:</b>
+		<p><form id="my_form_2" name="search_2" method="get" action="http://sc.it-academy.by/fe/form.php">
+		<fieldset>
+		<legend>Строка поиска</legend>
+		<input type="text" name="search" required autofocus> <span>*</span>
+		<input type="submit" value="Искать">
+		<br>
+		<label> <input type="radio" name="radio_name_2" value="site" id="checkbox_group_2">на сайте
+		</label>
+		<label> <input type="radio" name="radio_name_2" value="google" id="checkbox_group_2" checked>в google
+		</label>
+		</fieldset>
+		</form>
+		</p>
+		</th>
+		</tr>
+		<tr>
+		<th align="left" valign="top">
+		<b>Простая форма авторизации:</b>
+		<p><form id="my_form_3" name="author" method="get" action="http://sc.it-academy.by/fe/form.php">
```

Comment:
you send password by method get. it is a joke?

Reply:
Fixed. Replaced by method post